frontend/src/data/variables/AcsPopulationProvider.test.ts

```diff
@@ -187,6 +192,56 @@ describe("AcsPopulationProvider", () => {
     );
   });
 
+  test("State and Race Breakdown with standard race filter", async () => {
+    const rawData = [
+      stateRow(AL, RACE, TOTAL, 2),
+      stateRow(AL, RACE, ASIAN, 2),
+      stateRow(NC, RACE, TOTAL, 20),
+      stateRow(NC, RACE, ASIAN, 5),
+      stateRow(NC, RACE, WHITE, 15),
+      // Non-standard, will be excluded from the non-standard filter
+      stateRow(NC, RACE, "White", 17),
+      stateRow(NC, RACE, NON_HISPANIC, 13),
+    ];
+
+    const datasetId = "acs_population-by_race_state_std";
+    dataFetcher.setFakeDatasetLoaded(datasetId, rawData);
+
+    let breakdowns = Breakdowns.forFips(new Fips(NC.code)).andRace();
```

Comment:
Could you just put the breakdowns directly in the MetricQuery? I don't think we need to reuse the same variable

Reply:
You say that now, wait till our formatter gets its hands on it :P 

Anyway I moved it directly to the MetricQuery